src/BenchmarkDotNet.Core/Toolchains/DotNetCli/DotNetCliGenerator.cs

```diff
@@ -91,7 +96,11 @@ protected override void Cleanup(ArtifactsPaths artifactsPaths)
                 }
                 catch (Exception) when (attempt++ < 5)
                 {
+#if UAP
```

Comment:
does the `#if !UAP` in the top of the file excludes a chance for this `#if UAP`?